hztb-rest-api/src/main/java/com/svs/hztb/api/gcm/model/notification/Notification.java

```diff
@@ -42,279 +46,283 @@
+
+		/**
+		 * Sets the badge property.
+		 */
+		public Builder badge(int value) {
+			badge = value;
+			return this;
+		}
+
+		/**
+		 * Sets the tag property.
+		 */
+		public Builder tag(String value) {
+			tag = value;
+			return this;
+		}
+
+		/**
+		 * Sets the color property in {@literal #rrggbb} format.
+		 */
+		public Builder color(String value) {
+			color = value;
+			return this;
+		}
+
+		/**
+		 * Sets the click action property.
+		 */
+		public Builder clickAction(String value) {
+			clickAction = value;
+			return this;
+		}
+
+		/**
+		 * Sets the body localization key property.
+		 */
+		public Builder bodyLocKey(String value) {
+			bodyLocKey = value;
+			return this;
+		}
+
+		/**
+		 * Sets the body localization values property.
+		 */
+		public Builder bodyLocArgs(List<String> value) {
+			bodyLocArgs = Collections.unmodifiableList(value);
+			return this;
+		}
+
+		/**
+		 * Sets the title localization key property.
+		 */
+		public Builder titleLocKey(String value) {
+			titleLocKey = value;
+			return this;
+		}
+
+		/**
+		 * Sets the title localization values property.
+		 */
+		public Builder titleLocArgs(List<String> value) {
+			titleLocArgs = Collections.unmodifiableList(value);
+			return this;
+		}
+
+		public Notification build() {
+			return new Notification(this);
+		}
+
+	}
+
+	private Notification(Builder builder) {
+		title = builder.title;
+		body = builder.body;
+		icon = builder.icon;
+		sound = builder.sound;
+		badge = builder.badge;
+		tag = builder.tag;
+		color = builder.color;
+		clickAction = builder.clickAction;
+		bodyLocKey = builder.bodyLocKey;
+		bodyLocArgs = builder.bodyLocArgs;
+		titleLocKey = builder.titleLocKey;
+		titleLocArgs = builder.titleLocArgs;
+	}
+
+	/**
+	 * Gets the title.
+	 */
+	public String getTitle() {
+		return title;
+	}
+
+	/**
+	 * Gets the body.
+	 */
+	public String getBody() {
+		return body;
+	}
+
+	/**
+	 * Gets the icon.
+	 */
+	public String getIcon() {
+		return icon;
+	}
+
+	/**
+	 * Gets the sound.
+	 */
+	public String getSound() {
+		return sound;
+	}
+
+	/**
+	 * Gets the badge.
+	 */
+	public Integer getBadge() {
+		return badge;
+	}
+
+	/**
+	 * Gets the tag.
+	 */
+	public String getTag() {
+		return tag;
+	}
+
+	/**
+	 * Gets the color.
+	 */
+	public String getColor() {
+		return color;
+	}
+
+	/**
+	 * Gets the click action.
+	 */
+	public String getClickAction() {
+		return clickAction;
+	}
+
+	/**
+	 * Gets the body localization key.
+	 */
+	public String getBodyLocKey() {
+		return bodyLocKey;
+	}
+
+	/**
+	 * Gets the body localization values list, which is immutable.
+	 */
+	public List<String> getBodyLocArgs() {
+		return bodyLocArgs;
+	}
+
+	/**
+	 * Gets the title localization key.
+	 */
+	public String getTitleLocKey() {
+		return titleLocKey;
+	}
+
+	/**
+	 * Gets the title localization values list, which is immutable.
+	 */
+	public List<String> getTitleLocArgs() {
+		return titleLocArgs;
+	}
+
+	@Override
+	public String toString() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "toString" is 14 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://ec2-52-23-152-124.compute-1.amazonaws.com:9000/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)